addons/web/static/lib/pdfjs/web/viewer.js

```diff
@@ -16860,7 +16860,7 @@ class BasePreferences {
         "disablePageLabels": false,
         "enablePermissions": false,
         "enablePrintAutoRotate": true,
-        "enableScripting": true,
+        "enableScripting": false,
```

Comment:
When we patch libraries in place, we typically add a comment like "Odoo customization" or something. This isn't perfect, but this increase the chances of not losing the custo when updating the lib.

Reply:
You can link the PR in the comment.